### PR TITLE
Adding Perp Trade Stream and Transfer in manageCollateral

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: proto/api.proto [hunk 1]
```diff
@@ -532,12 +532,12 @@ service Api {
 
   rpc GetPerpContracts(GetPerpContractsRequest) returns (GetPerpContractsResponse) {
     option (google.api.http) = {
-      get: "/api/v1/trade/perp/contracts"
+      get: "/api/v1/market/perp/contracts"
```
Comment (aspin) [inline proto/api.proto:535]: can you also update the version at the top of this file? says 1.1 right now, should be 1.6

File: proto/api.proto [hunk 2]
```diff
@@ -532,12 +532,12 @@ service Api {
...
     };
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       description: "Returns details of available perp contracts";
       summary: "details of available perp contracts";
-      tags: ["Perp"];
+      tags: ["Perp", "Market"];
```
Comment (aspin) [inline proto/api.proto:540]: Since you're adding "Market" here, add "Trade" to a bunch of the other endpoints?

**Review by aspin — APPROVED** [review]

File: proto/api.proto (hunk 1 above)
Comment (tinystarinagalaxy) [reply proto/api.proto:535]: done

File: proto/api.proto (hunk 2 above)
Comment (tinystarinagalaxy) [reply proto/api.proto:540]: done

**Pull request merged**